include/openssl/bio.h

```diff
@@ -879,6 +876,17 @@ OPENSSL_EXPORT int BIO_meth_set_puts(BIO_METHOD *method,
 // BIO_meth_get_puts returns |puts| function of |method|.
 OPENSSL_EXPORT int (*BIO_meth_get_puts(const BIO_METHOD *method)) (BIO *, const char *);
 
+
+// General No-op Functions [Deprecated].
+
+// BIO_set_write_buffer_size returns zero.
+//
+// TODO: Add |OPENSSL_DEPRECATED|. nginx defines -Wno-deprecated-declarations
```

Comment:
Do we have a issue for this and the other TODOs?

Reply:
Good call, I'll create one